src/test/file/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 include_directories(${GLIB_INCLUDES})
 link_libraries(${GLIB_LIBRARIES})
 add_executable(test-file test_file.c)
+add_compile_options(test-file )
```

Comment:
What's this line for? Is it to clear some default options?

Reply:
Oops, good catch. I think that must have been a stale unsaved buffer from when I was fixing the file test. (I'd started to add compiler flags to ignore warnings, but switched to using pragmas instead)